app/controllers/papers_controller.rb

```diff
@@ -16,6 +16,7 @@ def show
   # GET /papers/new
   def new
     @paper = Paper.new
+    @authors = 5.times {@paper.authors.build}
```

Comment:
Well...you don't even need the 2nd line to start with, so, no. The only line that matters is the `5.times { @paper.authors.build }`

The second line is just to capture the `@authors` variable in case you're using it in a view. But really, you shouldn't do that, because you can always get the authors from `@paper.authors`